src/hb.hh

```diff
@@ -246,7 +246,7 @@ struct _hb_alignof
 #endif
 
 
-#if defined(_WIN32) || defined(__CYGWIN__)
+#ifdef _WIN32
```

Comment:
I'm not sure about this one.  This one should be harmless, right?  What happens if one is building the uniscribe backend?  is that supported?

Reply:
I actually have no idea what uniscribe is or whether it's supported.  But I agree that it's harmless, so I don't care if you leave it in.  Should I add a commit my branch to do that, or will you take care of it?  I don't have much experience with PRs, so I'm not sure which way the workflow for you is easiest.